pandas/tests/base/test_sizeof.py

```diff
@@ -0,0 +1,22 @@
+from sys import getsizeof
+
+from pandas import (
+    DataFrame,
+    Series,
+)
+
+
+def test_sysof():
+    getsizeof(DataFrame)
```

Comment:
Could you `assert` the result of all these calls?

Reply:
I just used one of the existing tests (test_memory_usage) by simply adding more Series.
Thank you very much. 
I will keep an eye out if anything else is required.